source/extensions/filters/http/dynamic_forward_proxy/dynamic_forward_proxy_proto_descriptors.h

```diff
@@ -0,0 +1,15 @@
+#pragma once
+
+namespace Envoy {
+namespace Extensions {
+namespace HttpFilters {
+namespace DynamicForwardProxy {
+
+// This function validates that the message descriptors that
+// are referenced in Any messages are available in the descriptor pool.
+bool validateProtoDescriptors();
```

Comment:
Yeah I would just kill all of this. The filter code won't compile if it's missing so there is no chance of it getting elided from the build.

Reply:
SGTM, done